test-configs/eu/snap-config.json

```diff
@@ -79,23 +79,23 @@
     "partitions": {
         "/dev/loop6": {
             "name": "snap_pelion-edge_x1",
-            "size": "2.4G"
+            "size": "2576980378"
```

Comment:
Are these units correct? They look inconsistent with the other test configs.

Reply:
Good finding, didn't remember to do proper diff on the eu-config. It's now aligned.